seed/csharp-sdk/basic-auth/src/SeedBasicAuth/BasicAuth/BasicAuthClient.cs

```diff
@@ -17,14 +17,15 @@ public BasicAuthClient(RawClient client)
     /// <summary>
     /// GET request with basic auth scheme
     /// </summary>
-    public async Task<bool> GetWithBasicAuthAsync()
+    public async Task<bool> GetWithBasicAuthAsync(RequestOptions? options)
     {
         var response = await _client.MakeRequestAsync(
             new RawClient.JsonApiRequest
             {
-                BaseUrl = _client.Options.BaseUrl,
+                BaseUrl = options?.BaseUrl ?? _client.Options.BaseUrl,
```

Comment:
if we're passing in the entire `requestOptions`, can we move this logic into `rawclient`?

Reply:
Discussed offline - 

Now that we're refactoring the environment-based option as a simple BaseURL, we can actually apply all the options within the `RawClient` (we previously weren't able to without exposing the environment on the `RequestOptions` type).